packages/core/README.md

```diff
@@ -69,3 +71,32 @@ Call `initialize` method in your `AppDelegate.mm`.
 Even if your app do not use Bluetooth, you **MUST** include usage description on iOS side. For more details, please take a look inside the [native documentation](https://docs.drivequant.com/get-started-drivekit/ios#project-configuration)
 
 Our recommandation is to use [react-native-permissions](https://github.com/zoontek/react-native-permissions). You can find an implementation example in the [demo application inside this repository](../demo/App.tsx).
+
+### Common
+
+To finish the module's initialization, you need to :
+
+1. Specify your API key;
+2. Identify the driver with a unique ID.
+
+#### Specify your API key
+
+```typescript
+setApiKey(key: string): void
+```
+
+#### Specify your API key
+
+```typescript
+setUserId(userId: string): void
+```
+
+### Validation
+
+To validation that the initialization has been done successfully, please check your native logs and verify that you can see the following success message.
```

Comment:
Pas important mais je crois que c'est "To validate" et non "To validation"

Reply:
Ah non très important merci !